packages/runtime/test/transport/account.test.ts

```diff
@@ -218,3 +219,25 @@ describe("Un-/RegisterPushNotificationToken", () => {
         expect(result).toBeSuccessful();
     });
 });
+
+describe("CheckDeletionOfIdentity", () => {
+    test("check deletion of Identity that is not deleted", async () => {
+        const result = await sTransportServices.account.checkDeletionOfIdentity();
+        expect(result.isSuccess).toBe(true);
+        expect(result.value.isDeleted).toBe(false);
+        expect(result.value.deletionDate).toBeUndefined();
+    });
+
+    test("check deletion of Identity that has IdentityDeletionProcess with expired grace period", async () => {
+        const identityDeletionProcess = (await sTransportServices.identityDeletionProcesses.initiateIdentityDeletionProcess()).value;
+
+        await sleep(10000);
+
+        const result = await sTransportServices.account.checkDeletionOfIdentity();
+        expect(result.isSuccess).toBe(true);
+        expect(result.value.isDeleted).toBe(true);
+        expect(result.value.deletionDate).toBe(identityDeletionProcess.gracePeriodEndsAt);
+    });
+
+    // TODO: test for deleted Identity
```

Comment:
when you do this test you can see if syncDatawallet does exactly what we think it should do ..